_gabarits-jsonld/README.md

```diff
@@ -47,9 +49,19 @@ Saisissez des types supplémentaires correspondant au type particulier de l’é
 ### [_name_](https://schema.org/name)
 Saisissez le titre de l’événement.
 
+### _@id_
+Saisissez une URI (Uniform Resource Identifier) constituant un identifiant unique pour cet événement à l'intérieur du domaine du site web. L'URI de l'événement doit être différente de l'URL de la page web de l'événement, puisqu'il s'agit de deux [_choses_](https://schema.org/Thing) (ou _entités_) distinctes dans le contexte des données structurées Schema.org. Cette URI doit aussi être pérenne : elle ne doit pas changer et doit demeurer accessible dans le temps. 
+
+Dans le cas d'une série de représentations d'un même spectacle, en plus de l'URI de la série, une URI distincte devrait être assignée à chaque représentation à l'aide de l'[Extension pour les Séries d'événements](https://github.com/culturecreates/artsdata-data-model/blob/master/_gabarits-jsonld/Event/event_series.jsonld). Assurez-vous qu'aucun sous-événement ne se voit assigner la même l'URI que l'objet de type _EventSeries_.
+
+Si vous n'êtes pas en mesure d'attribuer une URI pérenne à chaque événement, incluant chaque représentation d'une série de représentations, alors il est préférable de ne pas utiliser la propriété _@id_.[^2]
+
+[^2]:À défaut de pouvoir générer une réelle URI autonome pour chaque événement (comme, par exemple, un permalien ou une PURL), il est possible de générer une URI fonctionnelle en ajoutant à la suite de l'URL de la page web une ancre ***#*** suivie d'une chaîne de caractère servant d'identifiant unique d'événement. Voici un exemple fictif : `https://diffuseurdespectacles.ca/evenements/nomduspectacle/#e1324`. Dans cet exemple, l'URL de la page web est `https://diffuseurdespectacles.ca/evenements/nomduspectacle/`, alors que la chaîne de caractères `e1324` représente l'identifiant unique d'une représentation du spectacle dans le calendrier du site web. N'importe quelle chaîne de caractères peut convenir du moment où elle est unique à l'intérieur du domaine du site web. Ce pourrait être un identifiant local propre au CMS du site web et désignant la représentation dans la base de données du système. Ce pourrait aussi être la date et l'heure de la représentation. Il n'est pas nécessaire qu'un point d'ancrage soit réellement présent dans le corps de la page web : du moment où le symbole dièse et une chaîne de caractères unique est ajoutée à l'URL de la page web, cela satisfait à l'exigence d'une URL canonique dans le contexte de Schema.org.
```

Comment:
When you write ":À défaut de pouvoir générer une réelle URI autonome pour chaque événement (comme, par exemple, un permalien ou une PURL)" it means to me that using PURL or permalink is better.  I can actually disagree.  I think using "#" with a fragment identifier is perfectly good when a URI in not already available. I would propose using "#" (hash) over creating PURLs because the webpage will be eventually taken down and no one will update the PURL to point to somewhere else like Artsdata where the URI may still be accessible. So adding an extra layer of redirect with PURL doesn't really help.  Same comment on permalinks.  I recommend removing the first part of the sentence and starting with your excellent explication of how to generate a URI using a hash with a fragment identifier.  Like this " Il est possible de générer une URI fonctionnelle en ajoutant..."

Reply:
- J'ai fait la suppression que tu as suggérée. 
- J'ai aussi modifié le texte pour employer le terme « carré » plutôt que « dièse » pour désigner le « croisillon ».
- J'ai inséré une note pour préciser que la chaîne de caractères introduite par le carré s'appelle un « identificateur de fragment » et j'ai mis un hyperlien vers Wikipedia.